src/dist.rs

```diff
@@ -315,13 +335,58 @@ impl Dist {
     }
 }
 
+#[cfg(feature = "sass")]
+fn sass(
+    static_dir: &std::path::Path,
+    dist_dir: &std::path::Path,
+    options: sass_rs::Options
+) -> Result<()> {
+    fn is_sass(path: &std::path::Path) -> bool {
+        matches!(
+            path.extension().map(|x| x.to_str()).flatten(),
+            Some("sass") | Some("scss")
+        )
+    }
+
+    fn should_ignore(path: &std::path::Path) -> bool {
+        path
+            .file_name()
+            .expect("static file ends by `..`")
+            .to_str()
+            .map(|x| x.starts_with("_"))
+            .unwrap_or(false)
+    }
+
+    log::trace!("Generating dist artifacts");
+    let walker = walkdir::WalkDir::new(&static_dir).into_iter();
+    for entry in walker {
+        let entry = entry.context(format!("cannot walk into directory `{}`", &static_dir.display()))?;
+        let source = entry.path();
+        let dest = dist_dir.join(source.strip_prefix(&static_dir).unwrap());
+        let _ = fs::create_dir_all(dest.parent().unwrap());
```

Comment:
oh this is so much better imo!! wdyt?

it's crystal clear to me

Reply:
Since an error is fatal here, make sense

Reply:
no i meant "source", "dest", ... I can read the code clearly and understand it

Reply:
ah yeah, it's really better